omrsigcompat/makefile

```diff
@@ -24,7 +24,7 @@ top_srcdir := ..
 include $(top_srcdir)/omrmakefiles/configure.mk
 
 MODULE_NAME := omrsig
-ARTIFACT_TYPE := c_shared
+ARTIFACT_TYPE := cxx_shared
```

Comment:
This has nothing to do with OpenXL; it's just an error that could be corrected separately.

Reply:
removed from this PR.